dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05305AddPushPublishFilterColumnTest.java

```diff
@@ -0,0 +1,46 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import com.dotmarketing.exception.DotDataException;
+import java.sql.SQLException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task05305AddPushPublishFilterColumnTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    private void dropColumn() throws SQLException {
+        final String dropColumnSQL = "ALTER TABLE publishing_bundle DROP COLUMN filter_key";
+        final DotConnect dotConnect = new DotConnect();
+        dotConnect.executeStatement(dropColumnSQL);
+    }
+
+    private boolean checkColumnExists(){
+        final String dropColumnSQL = "select id from publishing_bundle where filter_key = ''";
```

Comment:
I think on some db these strategies may break the connection, I think we can query the catalog to check metainfo to see if the filter_key exists